app/src/services/object.js

```diff
@@ -232,7 +232,8 @@ const service = {
         path: data.path,
         public: data.public,
         active: data.active,
-        updatedBy: data.userId ?? SYSTEM_USER
+        updatedBy: data.userId ?? SYSTEM_USER,
+        lastSyncedDate: data.lastSyncedDate ?? undefined
```

Comment:
As I'm thinking about this further, I'm wondering if the ?? undefined is even necessary, as the only time this attribute would show up is if it was passed into the data argument explicitly; and by virtue of that happening, it'd already either be undefined, or contain a value of some form.

Extra note: We need to add a new missing JSDoc line above L221 to talk about the optional lastSyncedDate argument.

Reply:
I think I might've been thinking of other less dynamically typed languages when I wrote in `undefined` - like, the kind where trying to use it would trigger some kind of null pointer exception.

Is there any value in leaving it there as a documentation thing? Like, making it obvious that it's intended behaviour? 🤔